tests/test_sheet_collector.py

```diff
@@ -11,6 +11,8 @@
 
 from sheetshuttle import sheet_collector
 
+from sheetshuttle import util
+
```

Comment:
This will likely cause a linter issue since it's being imported but not used. Please make sure to remove this line